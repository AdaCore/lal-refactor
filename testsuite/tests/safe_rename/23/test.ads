package Test is
   function Convert (A : String) return Float is (Float'Value (A));

   function Other_Convert (A : String) return Integer is (Integer'Value (A));

   A : Float := Convert ("3.3");
end Test;
