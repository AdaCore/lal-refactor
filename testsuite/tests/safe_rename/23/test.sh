#!/bin/sh
safe_rename -P test.gpr -S test.ads -L 2 -R 13 -N Other_Convert --algorithm analyse_ast
