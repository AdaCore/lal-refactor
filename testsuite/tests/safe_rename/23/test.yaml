description: Test that there is no conflict when renaming functions that do not have the same return type.
driver: shell_script
