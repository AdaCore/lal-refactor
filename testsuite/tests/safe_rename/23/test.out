# Renaming node test.ads:2:13: 
References:
Unit:test.ads
Line: 2
Source Location Range:
Start_Line: 2
Start_Column: 13
End_Line: 2
End_Column: 20
Line: 6
Source Location Range:
Start_Line: 6
Start_Column: 17
End_Line: 6
End_Column: 24

